README.md

```diff
@@ -12,20 +12,21 @@ The content of the generated file should be added to the appropriate `client_sec
 
 Building
 --------
+Windows users must ensure that the following value is set in Docker Engine > Edit JSON: “experimental: true”.
 
-When building the amps (`mvn clean package`) you should also include a combination of the following profiles (`-P`). If not include, the defaults are applied.
+When building the amps (`mvn clean package`) you should also include a combination of the following profiles (`-P`). If not included, the defaults are applied.
 
 **Platform**
 
-`community` Build amps for Alfresco Community (default)
+'local' Build amps for local dev/test only
 
-`enterprise` Build amps for Alfresco Enterprise
+'docker-end-to-end-setup' Build amps for future end-to-end tests
 
 **Version**
 
 `6` Build amps for version 6.x of Alfresco
 
```

Comment:
This section isn't needed any more. I cannot see any version profiles on within any pom.xml files in the project.

Reply:
Agreed - we should remove Versions section if no longer relevant, thanks.